README.md

```diff
@@ -18,6 +19,65 @@ Specifically, if it enables services to scale _during_ deployment.
 > [!TIP]
 > A dashboard combining CloudFormation events, ASG activity and ASG metrics is available here - https://metrics.gutools.co.uk/d/cdvsv1d6vhp1cb/testing-asg-rolling-update.
 
+## Summary
+Can `AutoScalingRollingUpdate` be used to deploy updates to an auto-scaling group? Yes.
+- If not using horizontal scaling, `AutoScalingRollingUpdate` can be used, with any configuration.
+- If using horizontal scaling, `AutoScalingRollingUpdate` with dynamic configuration, is a viable deployment strategy. However, deployment times are significantly increased in certain conditions. 
+
```

Comment:
I wonder if we should also explicitly mention the issues that we found with scaling during deployment here, as this is highlighted specifically in the `What is this repository for?` section above?

Reply:
Attempt to do this. I we can make it clearer by replicating https://docs.google.com/document/d/1nWPDD4R4TzqIUe8iX5-oZfiV_bz3V5-JWX2p47AqamM/edit#heading=h.i1h2867tl6sj as an observation. I'll do that separately.

Reply:
> Attempt to do this. I we can make it clearer by replicating https://docs.google.com/document/d/1nWPDD4R4TzqIUe8iX5-oZfiV_bz3V5-JWX2p47AqamM/edit#heading=h.i1h2867tl6sj as an observation. I'll do that separately.

I've been unable to reliably reproduce this scenario, so cannot add as an observation.